portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,34 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+function addRandomFact(){
+    const facts = 
+    ['Hannah has tried to learn arabic 10 times. She still does not know it.',
+    'Hannah was once a girl scout!',
+    'Hannah has broken 1 bone and it from falling off a swing set.',
```

Comment:
"and it came from" maybe?

Reply:
Changed the syntax up; i think its better now?

Reply:
I don't see the change on my end.

Reply:
LGTM!